versioned/transfer/src/main/java/org/projectnessie/versioned/transfer/ImportResult.java

```diff
@@ -18,6 +18,7 @@
 import org.immutables.value.Value;
 import org.projectnessie.versioned.persist.adapter.HeadsAndForkPoints;
```

Comment:
Should we remove this `import` too?

Reply:
Will go away later - needed for the javadoc (which isn't currently incorrect)